scripts/UI/HostButton.gd

```diff
@@ -0,0 +1,20 @@
+extends Node
+
+signal show_connect_screen
+
+# Called when the button is loaded.
+func _ready():
+	var myIP
+	# TODO: Is this reliable, and portable between OS?
+	for ip in IP.get_local_addresses():
```

Comment:
Seeing similar logic in this script and Connecting.gd, is there some way to keep all the logic together and then just tie the buttons to it? Seems like that would be cleaner than a script for each button, but idk if that's possible so I could be wrong

Reply:
Done. Moved to the network.gd script, which is globally accessible